src/test/java/se/sundsvall/messaging/integration/db/entity/HistoryEntityTest.java

```diff
@@ -0,0 +1,86 @@
+package se.sundsvall.messaging.integration.db.entity;
+
+import static java.time.temporal.ChronoUnit.SECONDS;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.within;
+
+import java.time.LocalDateTime;
+
+import org.junit.jupiter.api.Test;
+
+import se.sundsvall.messaging.model.MessageStatus;
+import se.sundsvall.messaging.model.MessageType;
+import se.sundsvall.messaging.test.annotation.UnitTest;
+
+@UnitTest
```

Comment:
Why is this annotation added here?
From what I can see, the only purpose whit this annotation is to add: `@ActiveProfiles("junit")`

If this was a SpringBooTest it could have had a purpose (I can't really see the gain though), but this test isn't a SpringBootTest